CIP-0036/README.md

```diff
@@ -266,8 +273,23 @@ Fund 10:
 Fund 11:
  - added the `deregistration` metadata format.
 
-## Copyright
+## Path to Active
 
-This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode)
+### Acceptance Criteria
+
+- [ ] Have this registration format supported in Catalyst infrastructure for two Catalyst funds.
+- [ ] Have this registration format supported by three wallets/tools.
+  - [cardano-signer](https://github.com/gitmachtl/cardano-signer)
+
+### Implementation Plan
+
+- [x] Authors to provide test vector file.
+  - See the [schema file](./schema.cddl).
+- [x] Authors to provide CDDL schema file.
+  - See [test vector file](./test-vector.md).
+- [ ] Authors to test this format for security and compatibility with existing Catalyst infrastructure.
```

Comment:
```suggestion
- [x] Authors to test this format for security and compatibility with existing Catalyst infrastructure.
```
I can confirm compatibility myself (I registered for & successfully voted in Fund 10 with CIP-36 compatible command line tooling) so maybe if @stevenj can confirm no known security holes then this box can be ticked as well.